### PR TITLE
Connections Alerting GA [DOC-809]

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
Added docs for connections alerting beta

### Merge timing
After eng for Connections Alerting is complete (which is TBD)

### Related issues (optional)
https://segment.atlassian.net/browse/DOC-809

### CODE REVIEW

File: src/connections/alerting.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+Connections Alerting allows Segment users to receive in-app, email, and Slack notifications related to the performance and throughput of an event-streaming connection. 
+
+> info "Connections Alerting is currently in beta"
+> This means that the Connections Alerting feature is in active development, and some functionality may change before it becomes generally available. During the beta, Connections Alerting supports event-streaming connections: [event streaming sources](/docs/connections/sources/#event-streams-sources) and [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations).
+
+To access Connections Alerting, select an event-streaming connection (like a web library source or cloud mode destination) and click the **Alerts** tab. 
+
+On the Alerts tab, you can create alerts and view all active alerts for this connection. You can only edit or delete the alerts that you've created.
```
Comment (stayseesong) [inline src/connections/alerting.md:13]:
```suggestion
On the Alerts tab, you can create alerts and view all active alerts for this connection. You can only edit or delete the alerts that you create.
```

File: src/connections/alerting.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Source volume alerts
+
+You can create an alert that notifies you when the volume of events received by your source in the last 24 hours changes beyond a percentage you set. For example, if you set a change percentage of 4%, you would be notified when your source ingests less than 96% or more than 104% of the typical event volume.
```
Comment (stayseesong) [inline src/connections/alerting.md:17]:
```suggestion
You can create an alert that notifies you when the volume of events received by your source in the last 24 hours changes beyond a percentage you set. For example, if you set a change percentage of 4%, Segment notifies you when your source ingests less than 96% or more than 104% of the typical event volume. 
```

File: src/connections/alerting.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+To access Connections Alerting, select an event-streaming connection (like a web library source or cloud mode destination) and click the **Alerts** tab. 
+
+On the Alerts tab, you can create alerts and view all active alerts for this connection. You can only edit or delete the alerts that you've created.
+
+## Source volume alerts
+
+You can create an alert that notifies you when the volume of events received by your source in the last 24 hours changes beyond a percentage you set. For example, if you set a change percentage of 4%, you would be notified when your source ingests less than 96% or more than 104% of the typical event volume. 
+
+To receive a source volume alert in a Slack channel, you must create a Slack webhook first. For more information about Slack webhooks, see the [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks){:target="_blank”} documentation.
```
Comment (stayseesong) [inline src/connections/alerting.md:19]:
```suggestion
To receive a source volume alert in a Slack channel, you must first create a Slack webhook. For more information about Slack webhooks, see the [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks){:target="_blank”} documentation.
```

File: src/connections/alerting.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+To create a source volume alert: 
+1. Open your workspace and navigate to the [event streaming source](/docs/connections/sources/#event-streams-sources) you'd like to configure alerts for.
```
Comment (stayseesong) [inline src/connections/alerting.md:22]:
```suggestion
1. Navigate to the [event streams source](/docs/connections/sources/#event-streams-sources) you'd like to configure alerts for. 
```

Comment (stayseesong) [reply src/connections/alerting.md:22]: Maybe you can also add steps for how to access the event streams source such as
1. Navigate to Connections > Sources and select the Event streams tab.
2. Select the source you'd like to configure alerts for

File: src/connections/alerting.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+You can create an alert that notifies you when the volume of events received by your source in the last 24 hours changes beyond a percentage you set. For example, if you set a change percentage of 4%, you would be notified when your source ingests less than 96% or more than 104% of the typical event volume. 
+
+To receive a source volume alert in a Slack channel, you must create a Slack webhook first. For more information about Slack webhooks, see the [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks){:target="_blank”} documentation.
+
+To create a source volume alert: 
+1. Open your workspace and navigate to the [event streaming source](/docs/connections/sources/#event-streams-sources) you'd like to configure alerts for. 
+2. Select the Alerts tab and click **Create alert**. 
+3. On the Create alert sidesheet, enter a percentage of source volume change that you'd like to be notified for. 
+4. Select one or more channels you'd like to receive alerts in.
```
Comment (stayseesong) [inline src/connections/alerting.md:25]:
```suggestion
4. Select one or more channels you'd like to receive alerts in. You can choose from:
```

File: src/connections/alerting.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+3. On the Create alert sidesheet, enter a percentage of source volume change that you'd like to be notified for. 
+4. Select one or more channels you'd like to receive alerts in. 
+  - **Email**: Select this to receive notifications at the provided email address. 
+  - **Slack**: Select this to send alerts to one or more channels in your workspace. 
+  - **In-app**: Select this to receive notifications in the Segment app. To view your notifications, select the bell next to your user icon in the Segment app. 
+5. Click **Save**.
+
+To make changes to a source volume alert that you created, select the icon in the Actions column for the alert and click **Edit**. 
+
+To delete a source volume alert that you created, select the icon in the Actions column for the alert and click **Delete**.
```
Comment (stayseesong) [inline src/connections/alerting.md:33]: do we need to include "that you created" for each of these lines? Do only the alerts you create show up, or do alerts that other users create also show up?

File: src/connections/alerting.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  - **In-app**: Select this to receive notifications in the Segment app. To view your notifications, select the bell next to your user icon in the Segment app. 
+5. Click **Save**.
+
+To make changes to a source volume alert that you created, select the icon in the Actions column for the alert and click **Edit**. 
+
+To delete a source volume alert that you created, select the icon in the Actions column for the alert and click **Delete**. 
+
+## Successful delivery rate alerts
+
+You can create an alert that notifies you when the volume of events successfully received by your destination in the last 24 hours falls below a percentage you set. For example, if you set a percentage of 99%, you would be notified if you destination had a successful delivery rate of 98% or below.
```
Comment (stayseesong) [inline src/connections/alerting.md:37]:
```suggestion
You can create an alert that notifies you when the volume of events successfully received by your destination in the last 24 hours falls below a percentage you set. For example, if you set a percentage of 99%, Segment notifies you if your destination had a successful delivery rate of 98% or below. 
```

File: src/connections/alerting.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+To make changes to a source volume alert that you created, select the icon in the Actions column for the alert and click **Edit**. 
+
+To delete a source volume alert that you created, select the icon in the Actions column for the alert and click **Delete**. 
+
+## Successful delivery rate alerts
+
+You can create an alert that notifies you when the volume of events successfully received by your destination in the last 24 hours falls below a percentage you set. For example, if you set a percentage of 99%, you would be notified if you destination had a successful delivery rate of 98% or below. 
+
+To receive a successful delivery rate alert in a Slack channel, you must create a Slack webhook first. For more information about Slack webhooks, see the [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks){:target="_blank”} documentation.
```
Comment (stayseesong) [inline src/connections/alerting.md:39]:
```suggestion
To receive a successful delivery rate alert in a Slack channel, you must first create a Slack webhook. For more information about Slack webhooks, see the [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks){:target="_blank”} documentation.
```

File: src/connections/alerting.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+To create a successful delivery rate alert: 
+1. Open your workspace and navigate to the [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations) you'd like to configure alerts for.
```
Comment (stayseesong) [inline src/connections/alerting.md:42]:
```suggestion
1. Navigate to the [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations) you'd like to configure alerts for. 
```

Comment (stayseesong) [reply src/connections/alerting.md:42]: Maybe you can also add steps for how to access the cloud-mode destinations

Comment (stayseesong) [reply src/connections/alerting.md:42]: is the user supposed to navigate to cloud-mode destinations? The link leads to event-streams destinations

File: src/connections/alerting.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+You can create an alert that notifies you when the volume of events successfully received by your destination in the last 24 hours falls below a percentage you set. For example, if you set a percentage of 99%, you would be notified if you destination had a successful delivery rate of 98% or below. 
+
+To receive a successful delivery rate alert in a Slack channel, you must create a Slack webhook first. For more information about Slack webhooks, see the [Sending messages using incoming webhooks](https://api.slack.com/messaging/webhooks){:target="_blank”} documentation.
+
+To create a successful delivery rate alert: 
+1. Open your workspace and navigate to the [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations) you'd like to configure alerts for. 
+2. Select the Alerts tab and click **Create alert**. 
+3. On the Create alert sidesheet, enter a percentage. You will receive events if your successful delivery rate falls below this percentage. 
+4. Select one or more channels you'd like to receive alerts in.
```
Comment (stayseesong) [inline src/connections/alerting.md:45]:
```suggestion
4. Select one or more channels you'd like to receive alerts in. You can choose from:
```

File: src/connections/alerting.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+1. Open your workspace and navigate to the [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations) you'd like to configure alerts for. 
+2. Select the Alerts tab and click **Create alert**. 
+3. On the Create alert sidesheet, enter a percentage. You will receive events if your successful delivery rate falls below this percentage. 
+4. Select one or more channels you'd like to receive alerts in. 
+  - **Email**: Select this to receive notifications at either the email address associated with your account or another email address that you enter into this field. 
+  - **Slack**: Select this and enter a Slack webhook URL and channel name to send alerts to a channel in your Slack workspace.
+  - **In-app**: Select this to receive notifications in the Segment app. To view your notifications, select the bell next to your user icon in the Segment app. 
+5. Click **Save**.
+
+To make changes to a successful delivery rate alert that you created, select the icon in the Actions column for the alert and click **Edit**.
```
Comment (stayseesong) [inline src/connections/alerting.md:51]: Same comment as above: do we need to include "that you created" for each of these lines? Do only the alerts you create show up, or do alerts that other users create also show up?

File: src/connections/alerting.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+3. On the Create alert sidesheet, enter a percentage. You will receive events if your successful delivery rate falls below this percentage. 
+4. Select one or more channels you'd like to receive alerts in. 
+  - **Email**: Select this to receive notifications at either the email address associated with your account or another email address that you enter into this field. 
+  - **Slack**: Select this and enter a Slack webhook URL and channel name to send alerts to a channel in your Slack workspace.
+  - **In-app**: Select this to receive notifications in the Segment app. To view your notifications, select the bell next to your user icon in the Segment app. 
+5. Click **Save**.
+
+To make changes to a successful delivery rate alert that you created, select the icon in the Actions column for the alert and click **Edit**. 
+
+To delete a successful delivery rate alert that you created, select the icon in the Actions column for the alert and click **Delete**.
```
Comment (stayseesong) [inline src/connections/alerting.md:53]: Same comment as above: do we need to include "that you created" for each of these lines? Do only the alerts you create show up, or do alerts that other users create also show up?

File: src/guides/how-to-guides/set-up-notifications-alerts.md [hunk 13]
```diff
@@ -4,20 +4,33 @@ title: How do we set up event-triggered notifications or alerts?
 
 Below you'll find a bunch of ways to set up notifications for yourself based on the data you're sending through Segment. 
 
-### Google Analytics custom alerts
+## Connections Alerting
+
+Connections Alerting allows Segment users to receive in-app, email, and Slack notifications related to the performance and throughput of an event-streaming connection.
+
+> info "Connections Alerting is currently in beta"
```
Comment (stayseesong) [inline src/guides/how-to-guides/set-up-notifications-alerts.md:11]:
```suggestion
> info "Connections Alerting is in beta"
```

File: src/guides/how-to-guides/set-up-notifications-alerts.md [hunk 14]
```diff
@@ -4,20 +4,33 @@ title: How do we set up event-triggered notifications or alerts?
...
+> This means that the Connections Alerting feature is in active development, and some functionality may change before it becomes generally available. During the beta, Connections Alerting supports event-streaming connections: [event streaming sources](/docs/connections/sources/#event-streams-sources) and [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations).
```
Comment (stayseesong) [inline src/guides/how-to-guides/set-up-notifications-alerts.md:12]:
```suggestion
> The Connections Alerting feature is in active development, and some functionality may change before it becomes generally available. During the beta, Connections Alerting supports event-streaming connections: [event streaming sources](/docs/connections/sources/#event-streams-sources) and [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations).
```

File: src/connections/alerting.md [hunk 15]
```diff
@@ -0,0 +1,53 @@
+---
+title: Connections Alerting
+beta: true
+---
+
+Connections Alerting allows Segment users to receive in-app, email, and Slack notifications related to the performance and throughput of an event-streaming connection. 
+
+> info "Connections Alerting is currently in beta"
```
Comment (stayseesong) [inline src/connections/alerting.md:8]:
```suggestion
> info "Connections Alerting is in beta"
```

File: src/connections/alerting.md [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+> This means that the Connections Alerting feature is in active development, and some functionality may change before it becomes generally available. During the beta, Connections Alerting supports event-streaming connections: [event streaming sources](/docs/connections/sources/#event-streams-sources) and [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations).
```
Comment (stayseesong) [inline src/connections/alerting.md:9]:
```suggestion
> The Connections Alerting feature is in active development, and some functionality may change before it becomes generally available. During the beta, Connections Alerting supports event-streaming connections: [event streaming sources](/docs/connections/sources/#event-streams-sources) and [cloud-mode destinations](/docs/connections/destinations/#event-streams-destinations).
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/alerting.md (hunk 6 above)
Comment (forstisabella) [reply src/connections/alerting.md:33]: Alerts that other users create do show up, but you can only edit/delete those that you created.

**Review by stayseesong — APPROVED** [review]

File: src/connections/alerting.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
+---
+title: Connections Alerting
+beta: true
+---
+
+Connections Alerting allows Segment users to receive in-app, email, and Slack notifications related to the performance and throughput of an event-streaming connection. 
+
+> info "Connections Alerting is in beta"
+> The Connections Alerting feature is in active development, and some functionality may change before it becomes generally available. During the beta, Connections Alerting supports event-streaming connections: [event streaming sources](/docs/connections/sources/#event-streams-sources) and [cloud-mode destinations](/docs/connections/destinations/#:~:text=Cloud%2Dmode%3A%20The%20sources%20send%20data%20directly%20to%20the%20Segment%20servers%2C%20which%20then%20translate%20it%20for%20each%20connected%20downstream%20destination%2C%20and%20send%20it%20on.).
```
Comment (forstisabella) [inline src/connections/alerting.md:9]:
```suggestion
```

**Pull request merged**